components/TopNav.vue

```diff
@@ -1,16 +1,32 @@
 <template>
-  <div class="p-4 border-b flex justify-between">
-    <div class="font-semibold text-xl text-gray-800">Find a Doc</div>
-    <div>
-      <a href="https://ko-fi.com/Z8Z14YNUR" target="_blank"
-        ><img
-          height="36"
-          style="border: 0px; height: 36px"
-          src="https://cdn.ko-fi.com/cdn/kofi1.png?v=3"
-          border="0"
-          alt="Buy Me a Coffee at ko-fi.com"
-      /></a>
+    <div class="p-4 border-b flex justify-between">
+        <div class="font-semibold text-xl text-gray-800">Find a Doc</div>
+        <div>
+            <NuxtLink to="/">Home</NuxtLink>
+            <NuxtLink to="/aboutPage">About</NuxtLink>
+            <NuxtLink to="/newsPage">News</NuxtLink>
```

Comment:
```suggestion
           
```

Reply:
This link should be remove since the `newsPage` has been deleted.

Reply:
good find, updated